profiler.sh

```diff
@@ -86,7 +86,7 @@ function abspath() {
 
 
 OPTIND=1
-SCRIPT_DIR=$(dirname $0)
+SCRIPT_DIR=$(dirname "$(readlink -f "$0")")
```

Comment:
Have you checked this on macOS? AFAIK, `readlink -f` does not work there.

Reply:
damn... you're right the flag  `-f` not in the POSIX standard... sorry